conf/messages

```diff
@@ -1037,3 +1037,10 @@ updateCase.disclosureClosed.p2.part2 = , including the additional information an
 privateCitizen.customsDeclaration.title = Have you underpaid tax or duty on an import that was declared to customs?
 privateCitizen.customsDeclaration.heading = Have you underpaid tax or duty on an import that was declared to customs?
 privateCitizen.customsDeclaration.error.required = Select yes if you underpaid tax or duty on an import that was declared to customs
+
+# Private Citizen - Hand Off page
+privateCitizen.handOff.title = You cannot use this service
+privateCitizen.handOff.heading = You cannot use this service
+privateCitizen.handOff.p1 = This service can only be used where goods were originally declared to customs.
+privateCitizen.handOff.p2.1 = You need to contact us on the email address
+privateCitizen.handOff.p2.2 = and give brief details. We will take you through the process of paying the tax or duty that is owed.
```

Comment:
Could you change these to privateCitizen.handOff.p2.part1 and privateCitizen.handOff.p2.part2

We have used "part" previously on the project for p2's when there are more than one part